wazuh/config/entrypoint.sh

```diff
@@ -113,4 +113,16 @@ trap "ossec_shutdown; exit" SIGINT SIGTERM
 
 chmod -R g+rw ${DATA_PATH}
 
+##############################################################################
+# Interpret any passed arguments (via docker command to this entrypoint) as
+# paths or commands, and execute them.
+#
+# This can be useful for actions that need to be run before the services are
+# started, such as "/var/ossec/bin/ossec-control enable agentless".
+##############################################################################
+for CUSTOM_COMMAND in "$@"
+do
```

Comment:
Hi @forana
I have tested it and works perfectly. Great job.
The only modification I think this needs, is printing some extra information about what commands are being executed, to make it more visible in the docker logs, for example adding this code.

```suggestion
do
  echo "Executing command \`${CUSTOM_COMMAND}\`"
  exec_cmd_stdout "${CUSTOM_COMMAND}"
done
```
Would make the output look like this:
```
wazuh_1 | Executing command `echo hello world`:
wazuh_1 | hello world
wazuh_1 | Executing command: `ls /var/ossec/`:
wazuh_1 | api
wazuh_1 | api-template
wazuh_1 | backup
wazuh_1 | bin
wazuh_1 | data
wazuh_1 | etc
wazuh_1 | etc-template
wazuh_1 | framework
wazuh_1 | integrations
wazuh_1 | lib
wazuh_1 | logs
wazuh_1 | logs-template
wazuh_1 | queue
wazuh_1 | queue-template
wazuh_1 | ruleset
wazuh_1 | ruleset-template
wazuh_1 | stats
wazuh_1 | stats-template
wazuh_1 | tmp
wazuh_1 | var
wazuh_1 | wodles
```
This solution can't print characters like ' or " without scaping then with \\, but this way we can know which command the output corresponds to.

Thank you very much for developing this new feature.
We appreciate all the contributions that help us improve our code.

Best regards,
Miguel

Reply:
That makes sense to me. I had it in there originally and removed it after some internal waffling - I've applied your suggestion to add it back.